tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\ClassMethod\RemoveDefaultGetBlockPrefixRector\Fixture;
+
+use Symfony\Component\Form\AbstractType;
+
+class IconType extends ChoiceType
+{
+    public function getBlockPrefix()
+    {
+        return 'icon';
+    }
+}
+
+class ChoiceType extends AbstractType
+{
+    public function getBlockPrefix()
+    {
+        return 'choice';
```

Comment:
In SF 3.4, the parent (ChoiceType) **has** this method. [ref](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php#L357-L360)

So If I remove the method in **my** type, I will end up with "choice" and not the default implement in https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/AbstractType.php#L53-L56